### ISSUE TITLE
Cannot install locally with pip3

### ISSUE THREAD

**Issue by Capt0rr** [tobimensch/termsql#13]
Looks like some files are hard-coded to go into root?

Same thing if I use `--prefix` flag.

```
+pechkin ~ % pip3 install --user termsql
Collecting termsql
  Using cached https://files.pythonhosted.org/packages/97/cc/5fb0acdaf28502784f59e18f1f93353be4bf2562bed55d0750d6a4188b
b1/termsql-1.0.tar.gz
Requirement already satisfied: sqlparse in ./.local/lib/python3.6/site-packages (from termsql) (0.3.1)
Installing collected packages: termsql
  Running setup.py install for termsql ... error
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-dipxgdyz/te
rmsql/setup.py'"'"'; __file__='"'"'/tmp/pip-install-dipxgdyz/termsql/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', 
open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'
"'))' install --record /tmp/pip-record-35i_qrnx/install-record.txt --single-version-externally-managed --compile --user
 --prefix=
         cwd: /tmp/pip-install-dipxgdyz/termsql/
    Complete output (15 lines):
    running install
    running build
    running build_scripts
    creating build
    creating build/scripts-3.6
    copying and adjusting termsql -> build/scripts-3.6
    copying where -> build/scripts-3.6
    copying limit -> build/scripts-3.6
    copying groupby -> build/scripts-3.6
    copying orderby -> build/scripts-3.6
    copying select -> build/scripts-3.6
    changing mode of build/scripts-3.6/termsql from 644 to 755
    running install_data
    copying termsql.1.gz -> /usr/share/man/man1/
    error: could not create '/usr/share/man/man1/termsql.1.gz': Permission denied
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = 
'"'"'/tmp/pip-install-dipxgdyz/termsql/setup.py'"'"'; __file__='"'"'/tmp/pip-install-dipxgdyz/termsql/setup.py'"'"';f=g
etattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(com
pile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-35i_qrnx/install-record.txt --single-version-ex
ternally-managed --compile --user --prefix= Check the logs for full command output.
```

```
:pechkin ~ % pip3 --version
pip 19.2.3 from /usr/lib/python3.6/site-packages/pip (python 3.6)
:pechkin ~ % python3 --version 
Python 3.6.9
```

**Comment by tobimensch** [comment]
@3dc1d3 
The only hardcoded path I know is to the man page, and that's because there's no elegant way as far as I know that pip/setup.py provides for installing man pages.
If you can tell me how to do it, or provide a pull request/patch, I'll be happy, too.

**Comment by Capt0rr** [comment]
Unfortunately I know even less than you about this and googling didn't help much - I didn't expect this to be a python-wide issue, though I'm not surprised at this point.

My only suggestion is I assume the prefix path should be accessible by the setup script, so you could add it to the hard-coded path (and use a default value of '/' if it's unset).

Would that work?

**Comment by tobimensch** [comment]
Hmm, maybe your solution could work, I have too much on my hands right now to do it myself, if you want to you can make a pull request.
Man pages need to install system wide, at least when installed by root,
in other cases I'm not sure what should be done with man pages, since
they can't be accessed easily when they're not in the mandb, so you might
as well not install them.

Provide me with a nice patch, and I promise I'll merge it.

**Comment by Capt0rr** [comment]
The man pages should be installed into the same prefix as everything else - I have multiple non-python apps installed this way, and I'm pretty sure it's the same for all unix-style installs (autoconf etc) which respect the prefix arg.

You can add additional paths to the mandb, so it's not a problem to find them.  In any case it's better than not having them at all, or worse, a failed install over a trivial matter (even if the user has no intention of using the man pages).

https://github.com/tobimensch/termsql/pull/14

**Comment by tobimensch** [comment]
Thanks. Merged #14.
